### ISSUE TITLE
Is it a mistake on Gravity loss?

### ISSUE THREAD

**Issue by opendeeple** [isantesteban/snug#9]
Hi, In the paper, I have seen Gravity formula with negative mass. 
![image](https://user-images.githubusercontent.com/74002741/170714691-171a72b4-ee7a-4d9e-b657-8dd1afac5ad9.png)
When I compared it with your [gravitational_energy](https://github.com/isantesteban/snug/blob/main/losses/physics.py#L91) loss function I have seen with positive mass. 
```py
def gravitational_energy(x, mass, g=9.81, return_average=True):
  batch_size = tf.cast(tf.shape(x)[0], x.dtype)
  U = g * mass[tf.newaxis, tf.newaxis] * x[:, :, 1]

  if return_average:
    return tf.reduce_sum(U) / batch_size
  return tf.reduce_sum(U, axis=-1)
```
Can I use the gravitational_energy or need some changes? Thanks for your attention?

**Comment by isantesteban** [comment]
Hi! In the paper **g** is a vector with negative acceleration in the vertical direction, something like this [0.0, -9.81, 0.0]

The implementation defines **g** as a positive scalar and removes the negative sign, but the end result is the same. You can use the function as it is, sorry for the confusion!

**Comment by opendeeple** [comment]
Great, thanks for your answer!
